src/gui/properties/propertieswidget.h

```diff
@@ -55,15 +55,18 @@ class QAction;
 class QTimer;
 QT_END_NAMESPACE
 
-class PropertiesWidget: public QWidget, private Ui::PropertiesWidget
+class PropertiesWidget : public QWidget, private Ui::PropertiesWidget
```

Comment:
It seems that the Coding Guidlines do not say a word on this particular case. If so, it's up to you.